package.json

```diff
@@ -27,6 +27,7 @@
   },
   "homepage": "https://github.com/SonarSource/sonarjs-cli",
   "devDependencies": {
+    "@types/node": "^8.0.54",
```

Comment:
@inverno what for this dependency?

Reply:
@vilchik-elena ease of use of TS for the integration test